src/GraphNodeManagerViewExtension/GraphNodeManagerView.xaml

```diff
@@ -288,13 +243,31 @@
                     <DataGridTemplateColumn.CellTemplate>
                         <DataTemplate>
                             <StackPanel Orientation="Horizontal">
+                                <Grid>
+                                    <Image Source="/GraphNodeManagerViewExtension;component/Images/Union.png"
+                                       ToolTip="{x:Static p:Resources.Title_IsOutputSingleItem}"
+                                       Style="{StaticResource ImageStyle}"
+                                       Width="16"
+                                       Visibility="{Binding IsOutputSingleItem, Converter={StaticResource BooleanToVisibilityConverter}}"></Image>
+                                    <TextBlock 
+                                        ToolTip="{x:Static p:Resources.Title_TopLevelItemsNumber}"
```

Comment:
This is the place where we introduced a bug where we cant build this fork at all

Reply:
looks like it was my mistake @QilongTang, i didn't include the updated Resources.Designer.cs
should be working now